configs/openmrs_config/concepts/labs.csv

```diff
@@ -5,9 +5,9 @@ f52d731e-58da-4f3a-a480-77c5e9d8d2eb,,,Random Blood Sugar,រកកម្រិ
 1363AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,++,,++,,,,,,,,Misc,N/A,,,,
 1364AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,+++,,+++,,,,,,,,Misc,N/A,,,,
 1365AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,++++,,++++,,,,,,,,Misc,N/A,,,,
-f5f76f5e-2e6a-4112-ab4e-629299829ae3,,,Ketone,ពិនិត្យរកសារជាតិកេតូន,Ketone,ពិនិត្យរកសារជាតិកេតូន,,,,,,,Question,Coded,+;++;+++;++++,,,
+07015405-c865-4a22-8ad1-c8863d5f005b,,,Ketones (Urine),ពិនិត្យរកសារជាតិកេតូន (ក្នុងទឹកនោម),Ketones (Urine),ពិនិត្យរកសារជាតិកេតូន (ក្នុងទឹកនោម),,,,,,,Question,Coded,+;++;+++;++++,,,
 c423325e-be95-42e2-ae38-b940c2586327,,,Albumin (Urine),តេស្តអាលប៊ុយម៊ីន (ក្នុងទឹកនោម),Albumin (Urine),តេស្តអាលប៊ុយម៊ីន (ក្នុងទឹកនោម),0,90,mg/g,,,Yes,Test,Text,,,,
-07015405-c865-4a22-8ad1-c8863d5f005b,,,Ketones (Urine),ពិនិត្យរកសារជាតិកេតូន (ក្នុងទឹកនោម),Ketones (Urine),ពិនិត្យរកសារជាតិកេតូន (ក្នុងទឹកនោម),0,9,mmol/L,,,Yes,Test,Text,,,,
+f5f76f5e-2e6a-4112-ab4e-629299829ae3,,,Total Cholesterol,កូលេស្តេរ៉ុលសរុប,Total Cholesterol,កូលេស្តេរ៉ុលសរុប,0,9,mmol/L,,,Yes,Test,Text,,,,
```

Comment:
@kdaud , the concept provided here give a min of 0 and a max of 9.
Yet in the form it's 0-200. Which one is right? @kazlaw ?

Please align.

Reply:
> Yet in the form it's 0-200. Which one is right?

The range should be 0-200. I've updated the range and also the concept uuid in this [commit]([7641e7f](https://github.com/openmrs/ozone-distro-cambodia/pull/117/commits/7641e7f77008878e685b01ca28dd6743915d9203))